src/ngraph/frontend/onnx_import/wrappers/graph.hpp

```diff
@@ -16,14 +16,12 @@
 
 #pragma once
 
-#include <map>
```

Comment:
Please revert the change. This header file uses `map` and `ostream` explicitly. The header files MUST be self-contained (this is the rule). There's no guarantee that any of header file will provide the rest of header files you depend on implicitly (you should never think like that).